File: IWD_RTYÖ/dartsMain.js
async function main() {
    const readline = require('readline').createInterface({
        input: process.stdin,
        output: process.stdout
    });

    function kysy(question) {
        return new Promise(resolve => readline.question(question, resolve));
    }

    const LisaaPelaaja = require('./lisaaPelaaja');
    const PeliMode = require('./Pelimode');

    const peli = new LisaaPelaaja();

    let i = 0;
    while (i < 2) {
        const lisääUusi = await kysy("Anna pelaajan nimi: ");
        peli.lisaaUusipelaaja(lisääUusi, 0);
        i++;
    }

    console.log(peli.getKaikkiPelaajat());
    peli.haePelaaja();

    const valitsePelimode = new PeliMode();

    valitsePelimode.annaPelinPituus("l");
    console.log("\n*Lyhyt pelimuoto*\n" + valitsePelimode.toString());

    valitsePelimode.annaPelinPituus("p");
    console.log("\n*Pitkä pelimuoto*\n" + valitsePelimode.toString());

    readline.close();
}

main();

File: Ryhmätyö_kokeiluja_JAVALLA/Darts_peli/src/dartsPeli/dartsMain.java
package dartsPeli;

import java.util.Scanner;

public class dartsMain {
    public static void main(String[] args) {

        //Lisää pelaajat ja hae pelaaja(t).
        Scanner main = new Scanner(System.in);

        lisääPelaaja peli = new lisääPelaaja();

        int i = 0;
        while (i < 2) {
            System.out.println("Anna pelaajan nimi: ");
            String lisääUusi = main.nextLine();
            peli.lisääUusipelaaja(lisääUusi, 0);
            i++;
        }
        System.out.println(peli.getKaikkiPelaajat());
        peli.haePelaaja();


        //Valitse pelin pituus ("l" = lyhyt/301p, "p" = pitkä/501p)
        peliMode valitsePelimode = new peliMode();


        //"l" = lyhyt/301p
        valitsePelimode.annaPelinPituus("l");
        System.out.println("\n*Lyhyt pelimuoto*\n" + valitsePelimode);
        //"p" = pitkä/501p
        valitsePelimode.annaPelinPituus("p");
        System.out.println("\n*Pitkä pelimuoto*\n" + valitsePelimode);
    }
}
